modules/bpi/bpi.images.inc

```diff
@@ -213,3 +229,71 @@ function bpi_fetch_image($filename, $image_url) {
 
   return file_save_data($result->data, $filename, FILE_EXISTS_REPLACE);
 }
+
+/**
+ * Compute the image type from a field name on a node.
```

Comment:
Thanks for elaborating.

There are a few things that I find a bit unclear going through the codebase:

1. Even though we are working with assets instead of images there are places in the codebase that only support images e.g. the `<img/>` tag in `bpi_assets_form()`.
2. Functions, parameter names and comments - existing as well as new - refer to images instead of assets

The situation might occur because the service supports assets but the only assets we support currently are images. Also: I often read [image types as JPEG, PNG, GIF etc](http://php.net/manual/en/function.imagetypes.php).

I think it would be worth being more explicit about this in the codebase:

- Adding a sentence or two in the DocBlock about the asset types - in the file DocBlock or here
- Reviewing function names, parameters etc. for consistency. An example: This function is called `bpi_get_image_type()` but I think it would be a lot more clear if it was called `bpi_get_asset_type()`.